gpu/src/cuda/abstractions/state.rs

```diff
@@ -5,8 +5,7 @@ use cudarc::{
 };
 use std::sync::Arc;
 
-const STARK256_PTX: &str =
-    include_str!("../../../../math/src/gpu/cuda/shaders/fields/stark256.ptx");
+const STARK256_PTX: &str = include_str!("../../../../math/src/gpu/cuda/shaders/field/stark256.ptx");
```

Comment:
Maybe use `CARGO_MANIFEST_DIR` rather than count `../`?

Reply:
I knew there was a way to get that path! I couldn't find it, thank you!

Reply:
Hmm I'm having problems doing this, because `include_str!()` will only take string literals, I tried to use `concatcp!()` from `const_format` but it doesn't like that neither.